### ISSUE TITLE
Cannot find name 'VisibilityState'

### ISSUE THREAD

**Issue by michaelpumo** [vueuse/vueuse#1351]
### Describe the bug

Trying to run a build and getting the following error:

```
$ vue-tsc --noEmit && vite build

node_modules/@vueuse/core/index.d.ts:1153:82 - error TS2304: Cannot find name 'VisibilityState'.

1153 declare function useDocumentVisibility({ document }?: ConfigurableDocument): Ref<VisibilityState>;
                                                                                      ~~~~~~~~~~~~~~~
Found 1 error in node_modules/@vueuse/core/index.d.ts:1153
```

### Reproduction

https://stackblitz.com/edit/vitejs-vite-tulx4y?devtoolsheight=33&file=src/App.vue

### System Info

```Shell
System:
    OS: macOS 11.6.4
    CPU: (4) x64 Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz
    Memory: 700.77 MB / 8.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.13.0 - /usr/local/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 8.1.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 99.0.4844.51
    Firefox: 95.0.2
    Safari: 15.3
  npmPackages:
    @vueuse/core: ^7.7.0 => 7.7.0 
    vue: ^3.2.31 => 3.2.31
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vueuse/vueuse/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vueuse.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/vueuse/vueuse/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a VueUse issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to https://github.com/vuejs/core instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vueuse/vueuse/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by webfansplz** [comment]
Fixed #1337

**Comment by michaelpumo** [comment]
Thank you. Do you know if this will be in the next release?
